### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by gagb**
Updates and cleanups readme :)
- news was getting too long, push it down. Properly acknowledge other leads
- add a note from maintainers for contributors and users

Please feel free to provide feedback on wording, formatting, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,40 @@
...
+
+
+> *Note for contributors and users*</b>: [microsoft/autogen](aka.ms/autogen-gh) is the official repository of AutoGen project and it is under active development and maintenance under MIT license. We welcome contributions from developers and organizations worldwide. Our goal is to foster a collaborative and inclusive community where diverse perspectives and expertise can drive innovation and enhance the project's capabilities. We acknowledge the invaluable contributions from our existing contributors, as listed in [contributors.md](./CONTRIBUTORS.md). Whether you are an individual contributor or represent an organization, we invite you to join us in shaping the future of this project. For further information please also see [Microsoft open-source contributing guidelines](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing).
```
Comment (thinkall) [inline README.md:30]: The link "aka.ms/autogen-gh" doesn't work, it goes to "https://github.com/microsoft/autogen/blob/update-readme/aka.ms/autogen-gh". Maybe use "https://aka.ms/autogen-gh".

Comment (gagb) [reply README.md]: Sorry about this and thank you for catching it! I fixed it.

**Review by Hk669 — APPROVED** [review]
lgtm. thanks @gagb

**Review by ekzhu — APPROVED** [review]

**Review by jackgerrits — APPROVED** [review]

**Review by thinkall — APPROVED** [review]

**Review by afourney — COMMENTED** [review]
Jack mentioned using the blockquote notation to make the note more prominent: https://github.com/orgs/community/discussions/16925

I would be in favor of that.

**Comment by gagb** [comment]
> Jack mentioned using the blockquote notation to make the note more prominent: https://github.com/orgs/community/discussions/16925
> 
> I would be in favor of that.

Please feel free to push a commit!

**Review by LittleLittleCloud — APPROVED** [review]

**Review by ahmed-awadallah — APPROVED** [review]

**Review by rickyloynd-microsoft — APPROVED** [review]

**Review by DavidLuong98 — APPROVED** [review]

**Review by victordibia — APPROVED** [review]

**Pull request merged**